addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -316,10 +316,38 @@
             </div>
             <div t-if="attendees" class="row mb-3">
                 <div class="col-12 mb-2">
-                    <a class="btn btn-primary" title="Download All Tickets" target="_blank"
+                    <h5>Get your Tickets</h5>
+                </div>
+                <div class="col-12 mb-2 d-flex flex-column flex-md-row align-items-stretch align-items-md-baseline justify-content-md-start">
+                    <a class="btn btn-primary" title="Download All Tickets" target="_blank" style="min-width: 9rem;"
                         t-attf-href="/event/{{ event.id }}/my_tickets?registration_ids={{ attendees.ids }}&amp;tickets_hash={{ event._get_tickets_access_hash(attendees.ids) }}">
-                        Download Tickets <i class="ms-1 fa fa-download"/>
+                        Download <i class="ms-1 fa fa-download"/>
                     </a>
+                    <span class="mx-3 d-flex justify-content-center">or</span>
+                    <div class="o_wevent_send_by_email_widget ms-md-2 mt-2 mt-md-0" style="max-width: 32rem;">
+                        <div class="has-validation">
+                            <div class="input-group">
+                                <input id="o_wevent_send_by_email_input" type="text"
+                                    placeholder="john@doe.com" class="form-control"/>
+                                <button class="o_wevent_js_send_email btn btn-primary"
+                                    type="button"
+                                    t-att-data-event-id="event.id"
+                                    t-att-data-attendees-ids="attendees.ids"
+                                    t-att-data-tickets-hash="event._get_tickets_access_hash(attendees.ids)">
+                                    Send by Email
+                                </button>
+                            </div>
+                            <div class="invalid-feedback">
+                                Please enter valid email(s)
+                            </div>
```

Comment:
I see this is duplicated since we have two templates using it. Should we put it in a separate template and call it? what do you think?

(Note that if style were to be different, we could call it and have some variable for extra classes, like the t-set inside the t-call="website.record_cover" we can find in a few places)